python/mops/src/mops/reference_implementations.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from checks import check_hpe, check_sap, check_opsa, check_opsax, check_sasax
+
+
+def hpe(C, A, P):
```

Comment:
Please use long names for all the functions here

Reply:
Some of the full names are very long. I'm merging this branch and I'll change the names on the tests branch

Reply:
Yes they are long, but I would rather have something long and clear than short and obscure; especially for external users. We are not in 1966 using FORTRAN VI anymore 😄